package.json

```diff
@@ -47,7 +47,9 @@
   "dependencies": {
     "antd": "^4.20.1",
     "axios": "^0.27.2",
+    "bytes": "^3.1.2",
```

Comment:
undo changes to package.json and package-lock.json - these new packages are not in use in your code